bin/test-addon-pr.sh

```diff
@@ -52,48 +52,63 @@ check() {
   fi
 }
 
+modified_versions() {
+  local name=$1
+
+  # Get the version that's changed (filter out templates)
+  local versions=$(git diff --dirstat=files,0 "origin/${GITHUB_BASE_REF}" -- "addons/${name}" "origin/${GITHUB_BASE_REF}" -- "addons/${name}" | sed 's/^[ 0-9.]\+% addons\///g' | grep -v template | cut -f2 -d"/" | uniq |  sort -r )
+
+  echo $versions
+}
 
 check_addon() {
   local name=$1
 
   # Get the version that's changed (filter out templates)
-  local version=$(git diff --dirstat=files,0 "origin/${GITHUB_BASE_REF}" -- "addons/${name}" "origin/${GITHUB_BASE_REF}" -- "addons/${name}" | sed 's/^[ 0-9.]\+% addons\///g' | grep -v template | cut -f2 -d"/" | uniq |  sort -r | head -n 1)
+  local versions=$(modified_versions $name)
 
   # check if there is a valid version (files in the root don't count) & template files 
-  shopt -s nullglob
-  if [ -n "${version}" ] && compgen -G "./addons/$name/template/testgrid/*.yaml" > /dev/null; then
-    ADDON_AVAILBLE=true
-    echo "Found Modified Addon: $name-$version"
-  fi      
-  shopt -u nullglob     
+  for version in $versions
+  do
+    shopt -s nullglob
+    if [ -n "${version}" ] && compgen -G "./addons/$name/template/testgrid/*.yaml" > /dev/null; then
```

Comment:
Do we need to check `version` anymore? Assuming it always has a value in a bash loop.

Reply:
"minimal change" coding, will fix